minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Mixins/MixinMinecraftGameloop.java

```diff
@@ -268,14 +268,14 @@ private void runGameLoop() throws IOException
 
         this.mcProfiler.endSection(); //root
     }
-    
+
     @Overwrite
     public void setIngameFocus()
     {
         if (!this.inGameHasFocus) {
             this.inGameHasFocus = true;
             this.displayGuiScreen((GuiScreen) null);
-            this.leftClickCounter = 10000;
+            this.leftClickCounter = 0;
```

Comment:
If we leave this 10000 the agent has to wait that long before it is able to interact with anything again. Potentially a better fix would be to set this to 0 on respawn, but this should not affect the behavior of agents